yarn.lock

```diff
@@ -4126,6 +4126,11 @@ dayjs@^1.8.14:
   resolved "https://registry.yarnpkg.com/dayjs/-/dayjs-1.8.20.tgz#724a5cb6ad1f6fc066b0bd9a800dedcc7886f19e"
   integrity sha512-mH0MCDxw6UCGJYxVN78h8ugWycZAO8thkj3bW6vApL5tS0hQplIDdAQcmbvl7n35H0AKdCJQaArTrIQw2xt4Qg==
 
+dayjs@^1.8.28:
```

Comment:
Interesting, our yarn.lock has another declaration of `dayjs` just above this one.

Not sure if this is expected at all, or if we should de-dupe it?

Reply:
Interesting indeed, seems a long running issue with `yarn upgrade` and shared deps https://github.com/yarnpkg/yarn/issues/3967. I'll de-dupe this manually for now.